contractapi/contract_chaincode.go

```diff
@@ -311,6 +311,10 @@ func (cc *ContractChaincode) addContract(contract ContractInterface, excludeFunc
 		}
 	}
 
+	if len(ccn.functions) == 0 {
+		return fmt.Errorf("Contracts are required to have at least 1 chaincode callable method. Contract %s has none", ns)
```

Comment:
I'm not sold on this error message. what does "chaincode callable" mean? I might think my private method is "chaincode callable"? if I knew what it meant ;)

Reply:
Agreed